### ISSUE TITLE
[FEATURE] add appIsInstalled feature

### ISSUE THREAD

**Issue by pawellesniowski** [capacitor-community/file-opener#31]
Thank you SO much for your plugin it helped me a lot with PDF files, however, I have a big problem with opening .DOCX files. I am not able to use the Cordova file opener, so I gladly jumped on community version. 

Cordoba file-opener has a method to check whether we have an app to open .docx files:
capacitorFileOpener.appIsInstalled(userDocument.appId), then we install the appropriate app and we can open .docx files

In the capacitor-community file opener, we do not have the option to look for the app, 
Opening with the community file opener does not work eighter:
```
const  docksMimeType = "application/vnd.openxmlformats-officedocument.wordprocessingml.document"
FileOpener.open({ filePath: downloadedPath, contentType: docksMimeType, openWithDefault: false })
```

So question/feature request is:
How we can open .docx files with your plugin.

Thank you!

**Comment by ryaa** [comment]
There is no **appIsInstalled** method, similar to https://github.com/pwlin/cordova-plugin-file-opener2#fileopener2appisinstalledpackageid-callbackcontext right now, so this looks like this is a feature request.
PR is welcomed :)

**Comment by Galvictor** [comment]
pdf, docx and xlsx works fine but pptx is not working is give me an error:

![image](https://user-images.githubusercontent.com/12416921/222536226-64fc96ab-70c4-4eea-b6d5-f5d66341ec20.png)

**Comment by Galvictor** [comment]
Now it worked! I had to install a pptx viewer on my cellphone.

**Comment by Lyfei** [comment]
> There is no **appIsInstalled** method, similar to https://github.com/pwlin/cordova-plugin-file-opener2#fileopener2appisinstalledpackageid-callbackcontext right now, so this looks like this is a feature request. PR is welcomed :)

Have you considered developing this feature now?

**Comment by ryaa** [comment]
> Have you considered developing this feature now?

Yes. This is in my queue but i can not promise when it will be implemented.
